kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -356,7 +372,7 @@ define([
                     }
                     // treat messages relating to single jobs as if they were for a job list
                     // eslint-disable-next-line no-case-declarations
-                    const jobIdList = msgData.job_id ? [msgData.job_id] : msgData.job_id_list;
+                    const jobIdList = msgData[JOB_ID] ? [msgData[JOB_ID]] : msgData[JOB_ID_LIST];
```

Comment:
the `JOB_ID` and `JOB_ID_LIST` vars are defined at the top of the file so may as well use 'em!